how-to-use-azureml/automated-machine-learning/regression/auto-ml-regression.yml

```diff
@@ -0,0 +1,9 @@
+name: auto-ml-regression
```

Comment:
This file is not in AzureMlCli and should not be pushed to GitHub.
It is generated as part of running the associated notebook.